### ISSUE TITLE
Right align in sample invoice 

### ISSUE THREAD

**Issue by davidheijkamp** [Kozea/WeasyPrint#751]
In the [sample invoice](https://weasyprint.org/samples/invoice/invoice.pdf) the invoice number doesn't correctly align to the right, like the rest of the content. This seems to be related to the PDF rendering by weasyprint, because the source HTML + CSS renders the text correctly.

**Comment by liZe** [comment]
> This seems to be related to the PDF rendering by weasyprint, because the source HTML + CSS renders the text correctly.

You're right. The space between the first `dd` and the second `dt` is included when calculating the `dd`'s text position to align it right.

**Comment by liZe** [comment]
Sample to reproduce:
```html
<style>
  dl {
    text-align: right;
  }
  dt, dd {
    display: inline;
    margin: 0;
  }
  dt::before {
    content: '';
    display: block;
  }
</style>
<dl>
  <dt>Test</dt>
  <dd>12345</dd>
  <dt>Test</dt>
  <dd>12345</dd>
</dl>
```
